js/src/forum/index.js

```diff
@@ -20,31 +20,22 @@ extend(ForumApplication.prototype, 'mount', function() {
   }
 });
 
-m.route.mode = 'pathname';
-
-override(m, 'route', function(original, root, arg1, arg2, vdom) {
-  if (arguments.length === 1) {
-
-  } else if (arguments.length === 4 && typeof arg1 === 'string') {
-
-  } else if (root.addEventListener || root.attachEvent) {
-    root.href = vdom.attrs.href.replace('/embed', '/d');
-    root.target = '_blank';
-
-    // TODO: If href leads to a post within this discussion that we have
-    // already loaded, then scroll to it?
-    return;
-  }
-
-  return original.apply(this, Array.prototype.slice.call(arguments, 1));
+m.route.prefix = '';
```

Comment:
Is this actually still needed? I was able to remove it in my WP extension in Mithril2 and I'm not seeing any issue. I'm pretty sure Flarum sets it to empty prefix somewhere else already.

Reply:
Good point, it's not! Since this is working off of the Flarum app, and we have the empty prefix in the main ForumApplication, we should be fine without it.